docs/data-sources/ipam_subnets.md

```diff
@@ -0,0 +1,773 @@
+- `source` (String) The resource identifier.
+
+Read-Only:
+
+- `display_name` (String) The human-readable display name for the object referred to by _source_.
+- `value` (Number) The inherited value.
+
+
+<a id="nestedatt--results--inheritance_sources--asm_config--min_unused"></a>
+### Nested Schema for `results.inheritance_sources.asm_config.min_unused`
+
+Optional:
+
+- `action` (String) The inheritance setting for a field.  Valid values are: * _inherit_: Use the inherited value. * _override_: Use the value set in the object.  Defaults to _inherit_.
+- `source` (String) The resource identifier.
+
+Read-Only:
+
+- `display_name` (String) The human-readable display name for the object referred to by _source_.
+- `value` (Number) The inherited value.
+
+
+
+<a id="nestedatt--results--inheritance_sources--ddns_client_update"></a>
+### Nested Schema for `results.inheritance_sources.ddns_client_update`
+
+Optional:
+
+- `action` (String) The inheritance setting for a field.  Valid values are: * _inherit_: Use the inherited value. * _override_: Use the value set in the object.  Defaults to _inherit_.
+- `source` (String) The resource identifier.
+
+Read-Only:
+
+- `display_name` (String) The human-readable display name for the object referred to by _source_.
+- `value` (String) The inherited value.
+
+
+<a id="nestedatt--results--inheritance_sources--ddns_conflict_resolution_mode"></a>
+### Nested Schema for `results.inheritance_sources.ddns_conflict_resolution_mode`
+
+Optional:
+
+- `action` (String) The inheritance setting for a field.  Valid values are: * _inherit_: Use the inherited value. * _override_: Use the value set in the object.  Defaults to _inherit_.
+- `source` (String) The resource identifier.
+
+Read-Only:
+
+- `display_name` (String) The human-readable display name for the object referred to by _source_.
+- `value` (String) The inherited value.
+
+
+<a id="nestedatt--results--inheritance_sources--ddns_enabled"></a>
+### Nested Schema for `results.inheritance_sources.ddns_enabled`
+
+Optional:
+
+- `action` (String) The inheritance setting for a field.  Valid values are: * _inherit_: Use the inherited value. * _override_: Use the value set in the object.  Defaults to _inherit_.
+- `source` (String) The resource identifier.
+
+Read-Only:
+
+- `display_name` (String) The human-readable display name for the object referred to by _source_.
+- `value` (Boolean) The inherited value.
+
+
+<a id="nestedatt--results--inheritance_sources--ddns_hostname_block"></a>
+### Nested Schema for `results.inheritance_sources.ddns_hostname_block`
+
+Optional:
+
+- `action` (String) The inheritance setting.  Valid values are: * _inherit_: Use the inherited value. * _override_: Use the value set in the object.  Defaults to _inherit_.
+- `source` (String) The resource identifier.
+- `value` (Attributes) (see [below for nested schema](#nestedatt--results--inheritance_sources--ddns_hostname_block--value))
+
```

Comment:
We should start adding the support for querying inheritance in the read/list API for objects that support Inheritance

Reply:
We have a different story/bug for it. We can pick it up soon.